bundles/org.openhab.binding.sbus/src/main/java/org/openhab/binding/sbus/internal/config/SbusBridgeConfig.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.sbus.internal.config;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+import ro.ciprianpascu.sbus.Sbus;
+
+/**
+ * The {@link SbusBridgeConfig} class contains fields mapping bridge configuration parameters.
+ *
+ * @author Ciprian Pascu - Initial contribution
+ */
+@NonNullByDefault
+public class SbusBridgeConfig {
```

Comment:
Many of the condfiguration options listed in the bridge-udp.xml are not used in the binding? or not reflected here.

Reply:
Either implement (and document) them or remove them if not used.